Quake/gl_vidsdl.c

```diff
@@ -3844,7 +3848,7 @@ void M_Video_Draw (cb_context_t *cbx)
 
 		M_Mouse_UpdateCursor (&video_options_cursor, 12, 400, y, 8, i);
 		if (video_options_cursor == VID_OPT_PADDING)
-			video_options_cursor = VID_OPT_VSYNC;
+			video_options_cursor--;
```

Comment:
(just tweaking this to be more consistent with the adjustment elsewhere and to be less dependent on the specific menu items)